linux/CGroupMem.c

```
@@ -0,0 +1,236 @@
+   if (data->active && CGroupMem_readFile(nodeDir, "memory.swap.max", buf, sizeof(buf))) {
+      unsigned long long kb;
+      if (CGroupMem_parseLimit(buf, &kb)) {
+         data->swapLimit = kb;
+         data->swapActive = true;
+         if (CGroupMem_readFile(nodeDir, "memory.swap.current", buf, sizeof(buf)))
+            CGroupMem_parseUsage(buf, &data->swapCurrent);
+      }
+   }
+}
+
+bool CGroupMem_parseMountinfo(const char* content, char* mountBuf, size_t mountSize) {
```

Cppcheck (2.21.0)

[style] The function 'CGroupMem_parseMountinfo' should have static linkage since it is not used outside of its translation unit.

(staticFunction)